.github/workflows/auto-bump-datadog-agent.yml

```diff
@@ -0,0 +1,68 @@
+name: Open datadog-agent PR
+run-name:
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  open_bump_pr:
+    if: github.event.pull_request.merged == true && !contains(github.event.pull_request.labels.*.name, 'no-auto-bump')
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
Amazing, you also thought about a way to skip creating the PR.

Reply:
Indeed, it can be useful. This is highly inspired by @chouquette work on omnibus ruby 🙇